packages/components/app/styles/components/form/optional-indicator.scss

```diff
@@ -0,0 +1,10 @@
+//
+// FORM > OPTIONAL INDICATOR
+//
+// properties within each class are sorted alphabetically
+//
+
+.hds-form-optional-indicator {
+  color: var(--token-color-foreground-faint);
+  font-weight: var(--token-typography-font-weight-regular);
+}
```

Comment:
I _think_ we've lost the styling that sets the sizing of the element (probably just the `line-height`? I remember I checked the size, but maybe I misremember).

Reason to have it, is that in some conditions the alignment may appear off:
<img width="1792" alt="screenshot_1621" src="https://user-images.githubusercontent.com/686239/177790695-39f9eaf0-5ba8-4f0b-a578-dfe07c94b3c2.png">